apps/core/scenes/search/SearchContext.js

```diff
@@ -93,6 +105,8 @@ type State = {|
   },
 |};
 
+const defaultNightsInDestinationFrom = '3';
```

Comment:
Maybe we could move these default values to `@margarita/config` (as the default sort or the result limits)

Reply:
Done